v4/plugins/eclipse/che-machine-exec/nightly/devfile.yaml

```diff
@@ -0,0 +1,25 @@
+schemaVersion: "2.0.0"
+metadata:
+  name: "Che Machine Exec"
+components:
+- name: terminal
+  attributes:
+    "app.kubernetes.io/name": che-terminal.eclipse.org
```

Comment:
I'm not sure about its usage and who is supposed to provision that attributes, but it's not big deal. It's here at least for time being as David and Mario provided